wikipedia-sentences-incremental/app.py

```diff
@@ -0,0 +1,112 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import os
+import itertools as it
+
+import click
+from jina.flow import Flow
+from jina import Document
+
+
+def config():
+    os.environ['JINA_DATA_FILE_1'] = os.environ.get(
+        'JINA_DATA_FILE_1', 'data/input-1.txt'
+    )
+    os.environ['JINA_DATA_FILE_2'] = os.environ.get(
+        'JINA_DATA_FILE_2', 'data/input-2.txt'
+    )
+    os.environ["JINA_WORKSPACE"] = os.environ.get("JINA_WORKSPACE", "workspace")
+
+    os.environ["JINA_PORT"] = os.environ.get("JINA_PORT", str(45678))
+    # os.environ["JINA_MAX_DOCS"] = os.environ.get("JINA_MAX_DOCS", str(100))
```

Comment:
Maybe it's better to delete this comment if we don't need it

Reply:
Done